tests/unit/s2n_config_test.c

```diff
@@ -978,5 +982,110 @@ int main(int argc, char **argv)
         }
     }
 
+    /* Test s2n_config_get_supported_groups */
+    {
+        /* Safety */
+        {
+            DEFER_CLEANUP(struct s2n_config *config = s2n_config_new(), s2n_config_ptr_free);
+            EXPECT_NOT_NULL(config);
+            uint16_t supported_groups[S2N_TEST_MAX_SUPPORTED_GROUPS_COUNT] = { 0 };
+            uint16_t supported_groups_count = 0;
+
+            int ret = s2n_config_get_supported_groups(NULL, supported_groups, s2n_array_len(supported_groups),
```

Comment:
```suggestion
            EXPECT_FAILURE_WITH_ERRNO(s2n_config_get_supported_groups(NULL, supported_groups, s2n_array_len(supported_groups), S2N_ERR_NULL);
```
Why not 🤔 ?

Reply:
It's because clang-format doesn't like the normal line wrap with this macro as suggested. It would have to look like this which is a bit strange:

```
EXPECT_FAILURE_WITH_ERRNO(s2n_config_get_supported_groups(NULL, supported_groups,
                                  s2n_array_len(supported_groups),&supported_groups_count),
        S2N_ERR_NULL);
```

But if you prefer the clang-format acceptable one more let me know.